src/Service/PartyHelper.php

```diff
@@ -209,4 +209,38 @@ function ($relation) {
 
         return ['complainants' => $complainants, 'counterparties' => $counterparties];
     }
+
+    public function getSortingRelevantComplainant(CaseEntity $case): string
+    {
+        $complainantTypes = $this->getComplainantPartyTypesByCase($case);
+        // Gets first entry in array
+        $relevantType = reset($complainantTypes);
```

Comment:
If you change the return value of `getComplainantPartyTypesByCase` as suggested you have to change this as well. This use hints that the return value should be changed.

Reply:
I dont see any suggested changes to `getComplainantPartyTypesByCase `. 

However the change of return value from just `string` too `?string` has been addressed in places `getSortingCounterpart` and ´getSortingComplainant´ are called.